release/pkg/assets_capd.go

```diff
@@ -65,7 +65,7 @@ func (r *ReleaseConfig) GetDockerAssets() ([]Artifact, error) {
 		ProjectPath:       capiProjectPath,
 		SourcedFromBranch: sourcedFromBranch,
 	}
-	artifacts := []Artifact{Artifact{Image: imageArtifact}}
+	artifacts := []Artifact{{Image: imageArtifact}}
```

Comment:
These all seem a bit out of place. Was this some kind of lint or formatting change?

Reply:
yes, not related to the code changes for the oidc update.

Reply:
That change is on main => https://github.com/aws/eks-anywhere/blob/main/release/pkg/assets_capd.go#L68

My rebase probably got messed up and didn't include those.